src/test/setupSafeTest.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const request = require('supertest')
+require('~/initialization/envSetup')
+const initialization = require('~/initialization/initialization')
+const logger = require('~/logger/logger')
+const { restrictOperations } = require('~/test/helpers')
+const { restrictedOperations } = require('~/test/test-consts')
+
+const connectToDatabase = async () => {
+  const dbUri = process.env.MONGODB_URL_READONLY_TEST
```

Comment:
It would be nice to add `process.env.MONGODB_URL_READONLY_TEST` to config in \src\configs\config.js  for consistency as the the setup is done within src
and import as  const { config: { MONGODB_URL } } = require('~/configs/config') then